native/packages/react-native-bpk-component-horizontal-nav/readme.md

```diff
@@ -43,10 +43,11 @@ export default class App extends Component {
 
 ### BpkHorizontalNav
 
-| Property            | PropType                              | Required | Default Value |
-| -----------         | ------------------------------------- | -------- | ------------- |
-| children            | node                                  | true     | -             |
-| spaceAround         | bool                                  | false    | false         |
+| Property            | PropType                                  | Required | Default Value |
+| -----------         | ----------------------------------------- | -------- | ------------- |
+| children            | node                                      | true     | -             |
+| selected            | number (array index of the selected item) | false    | null          |
```

Comment:
if nothing is passed will default to `null` will null be cast a 0 so the first item will be selected or nothing will be selected?

Reply:
Nothing will be selected. I'm tightening it up a bit though so that `selected` is required.